examples/test-client/src/apis/Identity.tsx

```diff
@@ -1,61 +1,86 @@
+  const { z, connected } = useParcnetClient();
+  const [commitmentV3, setCommitmentV3] = useState<bigint | undefined>(
+    undefined
+  );
+  const [commitmentV4, setCommitmentV4] = useState<bigint | undefined>(
+    undefined
+  );
+  const [publicKey, setPublicKey] = useState<string | undefined>(undefined);
 
   return !connected ? null : (
     <div>
       <h1 className="text-xl font-bold mb-2">Identity</h1>
       <div className="prose">
         <div>
           <p>
-            Getting the identity commitment is done like this:
+            Getting the v3 identity commitment is done like this:
             <code className="block text-xs font-base rounded-md p-2">
-              await z.identity.getIdentityCommitment();
+              await z.identity.getSemaphoreV3Commitment();
             </code>
           </p>
           <TryIt
             onClick={async () => {
               try {
-                const commitment = await z.identity.getIdentityCommitment();
-                setCommitment(commitment);
+                const commitment = await z.identity.getSemaphoreV3Commitment();
+                setCommitmentV3(commitment);
               } catch (e) {
                 console.log(e);
               }
             }}
-            label="Get identity commitment"
+            label="Get v3 identity commitment"
           />
-          {commitment && <p>Commitment: {commitment.toString()}</p>}
+          {commitmentV3 !== undefined && (
+            <p>Commitment: {commitmentV3.toString()}</p>
+          )}
         </div>
         <div>
           <p>
-            Getting the attested emails is done like this:
+            Getting the v4 identity commitment is done like this:
             <code className="block text-xs font-base rounded-md p-2">
-              await z.identity.getAttestedEmails();
+              await z.identity.getSemaphoreV4Commitment();
             </code>
           </p>
           <TryIt
             onClick={async () => {
               try {
-                const emails = await z.identity.getAttestedEmails();
-                setEmails(emails);
+                const commitment = await z.identity.getSemaphoreV4Commitment();
+                setCommitmentV4(commitment);
               } catch (e) {
                 console.log(e);
               }
             }}
-            label="Get attested emails"
+            label="Get v4 identity commitment"
           />
-          {emails.length > 0 && (
-            <pre className="whitespace-pre-wrap">
-              {JSON.stringify(emails, null, 2)}
-            </pre>
+          {commitmentV4 !== undefined && (
+            <p>Commitment: {commitmentV4.toString()}</p>
           )}
         </div>
+        <div>
+          <p>
+            Getting the v4 identity public key is done like this:
+            <code className="block text-xs font-base rounded-md p-2">
+              await z.identity.getSemaphoreV4PublicKey();
```

Comment:
i wonder if we can drastically simplify this and just change the function name or alias this function to `z.identity.getPublicKey()`.

I think most new developers might be confused by "semaphore" and "V4" and public key is familiar enough.

Reply:
I'll update this in the next PR.